godtools/App/Features/Dashboard/Presentation/Tools/Presentation/ToolsView.swift

```diff
@@ -100,6 +98,43 @@ struct ToolsView: View {
             viewModel.pageViewed()
         }
     }
+    
+    @ViewBuilder func filterSection(width: CGFloat) -> some View {
```

Comment:
Hey @rachaelblue instead of using @ViewBuilder here I would like a View to encapsulate this instead.  I think it's cleaner and more readable to use a separate View file and if we started adding additional ViewBuilder methods for subviews it could bloat this file.

Reply:
That makes sense.  I was thinking you didn't want so many layers of subviews, but were you more concerned about layers of view models?